generators/app/templates/_eslintrc.json

```diff
@@ -1,5 +1,7 @@
 {
-
-   "extends": "@silvermine/eslint-config/node"
-
+   "extends": "@silvermine/typescript-config/tsconfig.json",
+   "include": [
+      "src/**/*",
+      "test/**/*"
```

Comment:
Do these need to be limited to `*.js` and `*.ts` files?

Reply:
Sorry, I tried to comment before you started reviewing. Those changes were meant for the tsconfig.json file, not eslintrc.json.